packages/did/README.md

```diff
@@ -127,7 +116,7 @@ did:kilt:4rp4rcDHP71YrBNvDhcH5iRoM3YzVoQVnCZvQPwPom9bjo2e
 
 Here, there is no `light:` component, which indicates that the DID is a full DID and that the keys associated with it must not be derived from the DID identifier but must be retrieved from the KILT blockchain.
 
-Beyond an authentication key, and encryption key, and external services, a full DID also supports an **attestation key**, which must be used to write CTypes and attestations on the blockchain, and a **delegation key**, which must be used to write delegations on the blockchain.
+Beyond an authentication key and encryption key, a full DID also supports an **attestation key**, which must be used to write CTypes and attestations on the blockchain, and a **delegation key**, which must be used to write delegations on the blockchain.
 
 ### Creating and anchoring a full DID
```

Comment:
Q: Should we mention the deposit in this section and having to select a submitter? Or is this info too advanced for this readme?

Reply:
https://github.com/KILTprotocol/sdk-js/pull/415/commits/4dd6821326a6fad519be8e32d775074bbfb687df